stellarphot/core.py

```diff
@@ -62,3 +64,307 @@ def __init__(self, gain=1.0 * u.electron / u.adu,
         self.gain = gain
         self.read_noise = read_noise
         self.dark_current = dark_current
+
+
+class BaseEnhancedTable:
+    """
+    A base class to hold an `astropy.table.Table` data table and extend it
+    so that chosen columns can be accessed as attributes.  This base class
+    imposes the requirement that the table have columns corresponding to
+    attributes 'id', 'ra', and 'dec' attributes, as well as confirming
+    the 'ra' and 'dec' are in degrees.
+
+    Parameters
+    ----------
+
+    table_descript: `numpy.ndarray`
+        This is a 2-D numpy array where each row of the array is the
+        table column name, dtype, astropy unit (can be None), and
+        the associated attribute name (can also be None, in which case
+        tha column of the table has no associated instance attribute).
+        The description must include 'ra', 'dec', and 'id' attributes
+        among the listed attributes.
+
+    data: `astropy.table.Table`, optional
+        A table containing all the instrumental aperture photometry results
```

Comment:
This doesn't have to be a photometry at this stage, right?

Reply:
Whoops, cut/paste error.  Fixed.